### PR TITLE
Deprecate `multi01` and `arm01` clusters

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/test-platform @jeffdyoung @danilo-gemoli @deepsm007 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by droslean** [comment]
> @droslean Can we hold until `build10` is avliable? QE would like to migrate their work from `multi01` first: https://github.com/openshift-qe/baremetal-qe-infra
> 
> cc: @aleskandro

I don't see the point of it, since multi01 and arm01 are not functional. @aleskandro technically, your postsubmit is not working now.

**Comment by aleskandro** [comment]
again, if that's not possible, fair enough, but if there is any chance we can keep it until build10 is functional, that would be great

**Comment by droslean** [comment]
/hold

Holding until build10 is in place.

/cc @bear-redhat

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -713,12 +706,7 @@ secret_configs:
         email_field: email
         item: registry.redhat.io-pull-secret
         registry_url: registry.redhat.io
-  to:
-  - cluster_groups:
-    - non_app_ci_arm
-    name: pull-secret
-    namespace: openshift-config
-    type: kubernetes.io/dockerconfigjson
+  to: null
```
Comment (droslean) [inline core-services/ci-secret-bootstrap/_config.yaml:709]: This is coming from `cluster-init` tool. Looks like a bug.

**Pull request merged**